lib/sift/client.rb

```diff
@@ -67,94 +69,147 @@ def original_request
+    #   attributes to track.  This parameter must be specified.
+    #
+    # opts (optional)::
+    #   A Hash of optional parameters for the request --
+    #
+    #   :return_score::
+    #     If true, requests that the response include a score for this
+    #     user, computed using the submitted event.  See
+    #     https://siftscience.com/developers/docs/ruby/score-api/synchronous-scores
+    #
+    #   :abuse_types::
+    #     List of abuse types, specifying for which abuse types a
+    #     score should be returned (if scoring was requested).  By
+    #     default, a score is returned for every abuse type to which
+    #     you are subscribed.
+    #
+    #   :return_action::
+    #     If true, requests that the response include any actions
+    #     triggered as a result of the tracked event.
+    #
+    #   :return_workflow_status::
+    #     If true, requests that the response include the status of
+    #     any workflow run as a result of the tracked event.  See
+    #     https://siftscience.com/developers/docs/ruby/workflows-api/workflow-decisions
     #
-    # timeout (optional)
-    #   The number of seconds to wait before failing the request. By default this is
-    #   configured to 2 seconds (see above). This parameter is optional.
+    #   :timeout::
+    #     Overrides the timeout (in seconds) for this call.
     #
-    # path (optional)
-    #   Overrides the default API path with a different URL.
+    #   :api_key::
+    #     Overrides the API key for this call.
     #
-    # return_score (optional)
-    #   Whether the API response should include a score for this user. The score will
-    #   be calculated using the submitted event.  This feature must be
-    #   enabled for your account in order to use it.  Please contact
-    #   support@siftscience.com if you are interested in using this feature.
+    #   :version::
+    #     Overrides the version of the Events API to call.
     #
-    # return_action (optional)
-    #   Whether the API response should include an action triggered for this transaction.
+    #   :path::
+    #     Overrides the URI path for this API call.
     #
-    # == Returns:
-    #   In the case of an HTTP error (timeout, broken connection, etc.), this
-    #   method returns nil; otherwise, a Response object is returned and captures
-    #   the status message and status code. In general, you can ignore the returned
-    #   result, though.
+    # ==== Returns:
     #
-    def track(event, properties = {}, timeout = nil, path = nil, return_score = false, api_key = @api_key, return_action = false)
-      warn "[WARNING] api_key cannot be empty, fallback to default api_key." if api_key.to_s.empty?
-      api_key ||= @api_key
+    # In the case of an HTTP error (timeout, broken connection, etc.),
+    # this method returns nil; otherwise, a Response object is
+    # returned and captures the status message and status code. In
+    # general, you can ignore the returned result, though.
```

Comment:
I agree that the comment shouldn't include "In general, you can ignore the returned result, though" -- I'll remove.